build/lib/release.sh

```diff
@@ -333,7 +304,7 @@ function kube::release::create_docker_images_for_server() {
     local binary_dir="$1"
     local arch="$2"
     local binary_name
-    local binaries=($(kube::build::get_docker_wrapped_binaries "${arch}"))
+    local binaries=("$(kube::build::get_docker_wrapped_binaries "${arch}")")
```

Comment:
I think we were intentionally splitting here? were we?

Reply:
Yes, it is an array. I will modify again.